features/ccx-upgrades-inference/perform_inference.feature

```diff
@@ -90,3 +90,336 @@ Feature: Upgrade Risks Prediction inference - test well known values
+                      "type": "array"
+                    },
+                    "operator_conditions": {
+                      "title": "Degraded Operator Condition",
+                      "type": "array"
+                    }
+                  }
+                }
+              }
+            }
+          """
+      And The body of the response is the following
+          """
+            {
+              "upgrade_risks_predictors": {
+                "alerts":[
+                  {
+                    "name": "APIRemovedInNextEUSReleaseInUse",
+                    "namespace": "openshift-kube-apiserver",
+                    "severity": "critical"
+                  },
+                  {
+                    "name": "OtherAlert",
+                    "namespace": "openshift-kube-apiserver",
+                    "severity": "critical"
+                  }
+                ],
+                "operator_conditions":[]
+              }
+            }
+          """
+
+Scenario: Check Inference Service does not predict risk for 2 alerts with info severity
+    Given The CCX Inference Service is running on port 8000
+     When I request the upgrade-risks-prediction endpoint in localhost:8000 with JSON
+          """
+            {
+              "alerts": [
+                {
+                  "name": "APIRemovedInNextEUSReleaseInUse",
+                  "namespace": "openshift-kube-apiserver",
+                  "severity": "info"
+                },
+                {
+                  "name": "OtherAlert",
+                  "namespace": "openshift-kube-apiserver",
+                  "severity": "info"
+                }
+              ],
+              "operator_conditions": []
+            }
+          """
+     Then The status code of the response is 200
+      And The body of the response has the following schema
+          """
+            {
+              "required": [
+                "upgrade_risks_predictors"
+              ],
+              "type": "object",
+              "properties": {
+                "upgrade_risks_predictors": {
+                  "title": "Upgrade Risks Predictors",
+                  "type": "object",
+                  "properties": {
+                    "alerts": {
+                      "title": "Alerts",
+                      "type": "array"
+                    },
+                    "operator_conditions": {
+                      "title": "Degraded Operator Condition",
+                      "type": "array"
+                    }
+                  }
+                }
+              }
+            }
+          """
+      And The body of the response is the following
+          """
+            {
+              "upgrade_risks_predictors": {
+                "alerts":[],
+                "operator_conditions":[]
+              }
+            }
+          """
+
+  Scenario: Check Inference Service does not predict risk for 1 alert with critical severity
```

Comment:
I would put these steps under the [“Check Inference Service does not predict risk for less than 2 alerts”](https://github.com/RedHatInsights/insights-behavioral-spec/pull/422/files#diff-69b3171c96a06c457be4324f4e6211d7b711da3748bd890f1ea395843c749f91R145) as it is also a “1 alert only” test

Reply:
Done